source/deploy/mattermost-changelog.md

```diff
@@ -11,6 +11,106 @@ From Mattermost v9.2, this changelog summarizes updates for the latest cloud and
+ - Fixed an issue where a proxy link was copied instead of the original image link when copying a post that included an embedded image.
+ - Fixed an issue where the user status would incorrectly get stuck to online after the user closed the tab.
+ - Fixed an issue where on some servers the user could not see the member count in the **Browse Channels** dialog.
+ - Fixed an issue with inline display of WebP images accessed through the public-link feature.
+ - Fixed an issue where it wasn’t clear that the ``mmctl import process --bypass-upload --local`` doesn't work if the server is in High Availability.
+ - Fixed an issue where the user status would incorrectly be set to offline without checking for connections in other nodes in an High Availability cluster.
+ - Fixed a longstanding issue where the @mention auto-complete could erase post text following the auto-completed @mention.
+ - Fixed an issue with status management to avoid missing notifications.
+ - Fixed an issue where audit events were not added for OAuth logins.
+ - Fixed an issue with the error check in the message export process.
+ - Fixed an issue where pasting in the post text box could not always paste without formatting.
+ - Fixed some plugin settings with defaults not changing value.
+
+### config.json
+New setting options were added to ``config.json``. Below is a list of the additions and their default values on install. The settings can be modified in ``config.json``, or the System Console when available.
+
+#### Changes to all plans:
+ - Under ``ExperimentalSettings`` in ``config.json``:
+    - Added ``DisableWakeUpReconnectHandler`` to disable the wake up on reconnect handler.
+ - Removed ``SelfHostedPurchase`` setting.
+
+#### Changes to the Enterprise plan: 
+ - Under ``MetricsSettings`` in ``config.json``:
+    - Added a feature flag and a setting ``EnableClientMetrics`` to control new client performance metrics.
+    - Added a setting for notification metrics ``EnableNotificationMetrics``.
+ - Self-hosted system administrators can now configure all ``ExperimentalAuditSettings`` through the user interface in the ``System Console``. Cloud administrators can now change the ``AdvancedLoggingJSON`` value for the ``ExperimentalAuditSettings``. This is the only configuration that Cloud administrators are able to adjust. Feature flag ``ExperimentalAuditSettingsSystemConsoleUI`` must be enabled in order to leverage this new user interface.
+
+### Websocket Event Changes
+ - Changed the semantics of the ``mattermost_http_websockets_total`` metric to track all open WebSocket connections, regardless of whether they are authenticated.
+ - Added a ``origin_client`` label to the ``mattermost_http_websockets_total`` Prometheus metric.
+
+### Go Version
+ - v9.9 is built with Go ``v1.21.8``.
+
+### Open Source Components
+ - Removed ``@stripe/react-stripe-js`` and ``@stripe/stripe-js``, and added ``web-vitals`` at https://github.com/mattermost/mattermost.
+
+### Known Issues
```

Comment:
@amyblais can we add a know issue for https://mattermost.atlassian.net/browse/MM-58500 that is fixed in v9.9.1 that affects the notifications metrics delivery rate: 

```
- Inactive websockets were being counted as part of notifications delivery rate for web and desktop that caused an incorrect decrease in that metric in the Notifications Health dashboard. 
```

cc/ @devinbinnie does the above looks good to you?

Reply:
@amyblais  I would also like to add a link to the dashboard here as well when available in Grafana.com. We'll work on it this week so we have it before publishing the release notes